packages/hooks/src/useHasSwapFreeAccount.ts

```diff
@@ -0,0 +1,13 @@
+import { useStore } from '@deriv/stores';
+
+const useHasSwapFreeAccount = () => {
```

Comment:
I suggest leaving a documentation above the hook and explain about its functionality!

Reply:
sure